### DISCUSSION TITLE
What does "Torrents Space" mean?

### DISCUSSION THREAD

**Discussion by AyaanZaveri** [KiraLT/torrent-stream-server#256] [Q&A] [answered]
![Screen Shot 2021-06-15 at 2 15 22 PM](https://user-images.githubusercontent.com/63752541/122103029-20b67c80-cde4-11eb-8606-31f5d2721f44.png)

I've been using **Okteto**, and it's working great, but what happens when the **Torrents Space** reaches the max amount?
Also, why does it show **79.68 / 96.75** for disk space if I've not downloaded something?

**Comment by KiraLT** [comment, marked as answer]
Disk space shows the total space your server has. There will always be used space on server disk space because at least the operating system and other things are installed in the server.

Torrents space shows disk space only for a specific directory - where are torrents stored. If you reach 100% of the torrent disk space, your app will probably crash. However, it shouldn't happen - all inactive torrents are automatically deleted and the less space you have, the more aggressive this algorithm is.

**Reply by lu4t** [reply]
is there a way to manually clean the "torrent space"?

**Reply by KiraLT** [reply]
The process is automatic, but you can also delete `/tmp/torrent-stream-server` directory and restart the app.
